slatedb/src/db/builder.rs

```diff
@@ -191,6 +203,8 @@ impl<P: Into<Path>> DbBuilder<P> {
     /// Builds and opens the database.
     pub async fn build(self) -> Result<Db, SlateDBError> {
         let path = self.path.into();
+        // TODO: proper URI generation, for now it works just as a flag
```

Comment:
Making sure I follow the use as a flag. Is the purpose to disallow certain features (external gc, checkpoints/clones) if the user created the db with a wal object store?

Reply:
yes, exactly